data.md

```diff
@@ -192,7 +182,6 @@ Here we provide simple syntactic sugar over our power-modulus operator.
     syntax Int ::= powmod(Int, Int, Int) [function]
  // -----------------------------------------------
     rule powmod(W0, W1, W2) => W0 ^%Int W1 W2 requires W2 =/=Int 0
```

Comment:
This is ok because a rule for expmod in `iele.md` was the only caller of `powmod`, and it has conditions to make `0` an error rather than ever calling `powmod` with `W2` being 0.